.github/workflows/ci.yml

```diff
@@ -71,7 +71,7 @@ jobs:
 
       - name: SonarCloud
         if: runner.os == 'Linux'
-        uses: SonarSource/sonarcloud-github-action@v2.1.1
+        uses: SonarSource/sonarcloud-github-action@v2.3.0
```

Comment:
Not sure if you still wanna pin patch version or pin major versions which is done for the other actions

Reply:
I prefer to pin - less surprizes in wrong time.